deploy/cephfs/kubernetes/csi-cephfsplugin-provisioner.yaml

```diff
@@ -43,7 +43,7 @@ spec:
       priorityClassName: system-cluster-critical
       containers:
         - name: csi-cephfsplugin
-          image: quay.io/cephcsi/cephcsi:v3.11.0
+          image: quay.io/cephcsi/cephcsi:v3.11-canary
```

Comment:
same change is required in value.yaml for both helm charts can you please check?